examples/discovery-search-app/src/ExampleDocumentProvider.js

```diff
@@ -10,12 +10,31 @@
  */
 export class ExampleDocumentProvider {
   async provides(document: DocumentProviderProps) {
-    return document.extracted_metadata.filename === 'Art Effects.pdf';
+    // return document.extracted_metadata.filename === 'Art Effects.pdf';
+    console.log('dorian change', document.extracted_metadata.filename);
+    if (document.extracted_metadata.filename.indexOf('.pdf') >= 0) {
```

Comment:
I don't know if this difference would ever matter, but you can also use `document.extracted_metadata.file_type === 'pdf'` as your check here (and wherever else you want to verify the document type). Honestly, since you've worked on the backend of Answers, you might have a better idea of how reliable the filename of a document that gets returned to us is.

Reply:
I restored the original version. This was for local testing.